db/migrations/collector/20240109170034_fcd_tx_log.up.sql

```diff
@@ -0,0 +1,35 @@
+-- WRITE YOUR MIGRATION CODES FOR UP or DOWN
+
+-- Convention
+-- Write snake_case for your tables and columns
+
+-- Index name recommendation {tablename}_{columnname(s)}_{suffix} columnames should be alphabetical order
+-- {suffix}
+-- pkey for a Primary Key constraint
+-- key for a Unique constraint
+-- excl for an Exclusion constraint
+-- idx for any other kind of index
+-- fkey for a Foreign key
+-- check for a Check constraint
+BEGIN;
+
+-- Table Definition
+CREATE TABLE "public"."fcd_tx_log" (
+    "id" BIGSERIAL NOT NULL PRIMARY KEY,
+    "fcd_offset" int8 NOT NULL,
+    "height" int8 NOT NULL,
+    "timestamp" timestamp NOT NULL,
+    "hash" varchar NOT NULL,
+    "address" varchar NOT NULL,
+    "event_log" varchar NOT NULL,
+    "tx_index" int4,
+    PRIMARY KEY ("id")
```

Comment:
primary key has been declared twice

Reply:
https://github.com/dezswap/cosmwasm-etl/pull/14/commits/3b5801afbd299c57bcd2641f6030bebece49d67e

Thank you!